.github/workflows/build-ios-dev.yml

```diff
@@ -0,0 +1,74 @@
+name: IOS Development Build
+
+on:
+  workflow_dispatch:
+    inputs:
+      logLevel:
```

Comment:
Is that param used? On don't see any occurence in the yml file, but maybe it is used by a sub-action?

Reply:
It adds some GUI options on the manual workflow trigger. I can remove them from now to make it leaner